components/compliance-service/reporting/relaxting/reports.go

```diff
@@ -573,6 +573,45 @@ func (backend ES2Backend) getFiltersQuery(filters map[string][]string, latestOnl
 	boolQuery := elastic.NewBoolQuery()
 	boolQuery = boolQuery.Must(typeQuery)
 
+	filterTypes := []string{"environment", "organization", "chef_server", "chef_tags",
```

Comment:
I had to refactor this code because I was getting the below error.

`reporting/relaxting/reports.go:568:1: cyclomatic complexity 35 of func `(ES2Backend).getFiltersQuery` is high (> 30) (gocyclo)`